.travis.yml

```diff
@@ -54,13 +54,7 @@ before_script:
   - cd apps/$APP_NAME
 
 script:
-  - ./occ face:setup
-  - make test
-
-  # Create coverage report
-  - wget https://scrutinizer-ci.com/ocular.phar
-  - php ocular.phar code-coverage:upload --format=php-clover clover.xml
+  - bash ./tests/travis-script.sh
```

Comment:
I move everything within the same script to ensure run within the same 'context' or 'stage'